tools/build_third_party.py

```diff
@@ -26,5 +38,30 @@ def run(args):
     subprocess.check_call(args, env=env)
 
 
+def remove_and_symlink(target, name):
+    try:
+        os.unlink(name)
+    except:
+        pass
+    os.symlink(target, name)
```

Comment:
It seems like this line should be `symlink(target, name)`. Otherwise, the `symlink` function below is unused.

Reply:
Thanks for catching it. That seems fixed now in master https://github.com/ry/deno/blob/master/tools/util.py#L25